api/src/pcapi/routes/backoffice_v3/finance/blueprint.py

```diff
@@ -210,6 +225,51 @@ def get_incident(finance_incident_id: int) -> utils.BackofficeResponse:
                 offers_models.Offer.url,
                 offers_models.Offer.subcategoryId,
             ),
+            # collective booking infos
```

Comment:
```suggestion
            # collective booking info
```
En anglois, `information` au pluriel est un indénombrable, et ne prend donc pas de `s`

Reply:
Et pareil pour tous les autres `infos` ;)